model/src/cfd.rs

```diff
@@ -886,7 +886,7 @@ impl Cfd {
         )?;
 
         tracing::debug!(order_id = %self.id, rollover_version = %version,
-            interval_in_hours = %self.settlement_interval.whole_hours(), "Maker accepting a rollover proposal");
+            interval_in_hours = %self.settlement_interval.whole_hours(), "Accepting rollover proposal");
```

Comment:
Thank you!!! This was bugging me every time I read it because I thought it is emitted by the taker due to the use of "maker" in this message :joy: